### PR TITLE
More clearly cross-reference rlang_backtrace_on_error

### PR DESCRIPTION
**Pull request by hadley**
I couldn't re-build the docs because `load_all()` gives me:

```
Error in new_weakref(env, finalizer = ns_finalizer(dll_path)) : 
  object 'rlang_new_weakref' not found
Calls: suppressPackageStartupMessages ... load_code -> <Anonymous> -> load_dll -> new_weakref
Execution halted
```

### CODE REVIEW

File: R/cnd-abort.R [hunk 1]
```diff
@@ -38,24 +38,10 @@
...
-#'
-#' * `"reminder"`: Invite users to call `rlang::last_error()` to see a
-#'   backtrace.
-#' * `"branch"`: Display a simplified backtrace.
-#' * `"collapse"`: Display a collapsed backtrace tree.
-#' * `"full"`: Display a full backtrace tree.
-#' * `"none"`: Display nothing.
+#' `abort()` always saves a backtrace. You can print a simplified backtrace of
+#' the last error by calling [last_error()] and a full backtrace with
+#' `summary(last_error())`. Control the default behaviour with
```
Comment (lionel-) [inline R/cnd-abort.R:43]: This makes it seem like the error controls the default for `last_error()`. Perhaps:

> You can also print a backtrace immediately when an error is thrown. See [`rlang_backtrace_on_error`].

File: R/cnd-abort.R [hunk 2]
```diff
@@ -977,6 +963,8 @@ peek_backtrace_on_error <- function() {
 #' * `last_trace()` is a shortcut to return the backtrace stored in
 #'   the last error. This backtrace is printed in full form.
 #'
+#' See [`rlang_backtrace_on_error`] to control the default behaviour.
```
[the remark this answers is not in the document]
Comment (lionel-) [reply R/cnd-abort.R:950]: Same as above?

```suggestion
#' To print a backtrace immediately when an error is thrown, see [`rlang_backtrace_on_error`].
```

File: R/cnd-abort.R (hunk 1 above)
Comment (hadley) [reply R/cnd-abort.R:43]: That feels maybe a bit too specific? How about ``Learn how to control what's displayed when an error is thrown with [`rlang_backtrace_on_error`]`` ?

Comment (lionel-) [reply R/cnd-abort.R:43]: yup looks good

**Pull request merged**